core/src/wallet/ripple/RippleLikeAccount.cpp

```diff
@@ -461,5 +461,28 @@ namespace ledger {
             });
         }
 
+        void RippleLikeAccount::isAddressActivated(const std::string &address,
+                                                   const std::shared_ptr<api::BoolCallback> &isActivated) {
+            isAddressActivated(address).callback(getContext(), isActivated);
+        }
+
+        Future<bool> RippleLikeAccount::isAddressActivated(const std::string &address) {
+            auto xrpAddress = RippleLikeAddress::parse(address, getWallet()->getCurrency());
+            if (!xrpAddress) {
+                return Future<bool>::successful(false);
+            }
+            auto self = getSelf();
+            return _explorer->getBaseReserve().flatMap<bool>(getContext(), [self, xrpAddress](const std::shared_ptr<BigInt> &reserve) -> Future<bool> {
+                return self->_explorer->getBalance(
+                        std::vector<RippleLikeKeychain::Address>{
+                                std::dynamic_pointer_cast<RippleLikeAddress>(xrpAddress)
+                        }).map<bool>(self->getContext(), [reserve](const std::shared_ptr<BigInt> &balance) -> bool {
+                    if (!reserve || !balance) {
+                        return false;
+                    }
+                    return reserve->compare(*balance) < 0;
```

Comment:
Is it possible to add the global `operator<` to compare `BigInt` ? Like that we don't manage anymore the result value.

Reply:
Yes, please create a Jira ticket; this has nothing to do with the current PR and it would make it’s merge slower.